bedrock/base/templates/macros.html

```diff
@@ -290,7 +290,7 @@ <h3>{{ _('Your download link was sent.') }}</h3>
   </section>
 {%- endmacro %}
 
-{% macro fxa_create_account_button(button_location, id_prefix, campaign, entrypoint, content = '', source = '' ) %}
+{% macro fxa_create_account_button(button_location, id_prefix, campaign, entrypoint, button_text, button_class, content = '', source = '') %}
```

Comment:
To make these parameters optional I think they need to be in the same format as content and source.